health/api/src/test/scala/com/cerner/beadledom/health/dto/HealthDependencyDtoSpec.scala

```diff
@@ -53,4 +54,16 @@ class HealthDependencyDtoSpec extends FunSpec with MustMatchers {
       dto.getName must be(none)
     }
   }
+
+  val healthDependencyDto = HealthDependencyDto.builder()
+    .setHealthy(true)
+    .setId("1")
+    .setName("name")
+    .setMessage("message")
+    .setPrimary(true)
+    .setLinks(LinksDto.builder().setSelf("link").build())
+    .build()
+
+  it must behave like aJacksonSerializableObject(healthDependencyDto)
+
```

Comment:
nit: empty line here and at other places.

Reply:
Removed empty lines, https://github.com/cerner/beadledom/pull/100/commits/e90e469d9c9a4c65fb23ae4bbd01e37a90a39527